flake.nix

```diff
@@ -0,0 +1,49 @@
+{
+  description = "Cubical Agda";
+
+  inputs.flake-utils.url = "github:numtide/flake-utils";
+
+  outputs = { self, flake-utils, nixpkgs }:
+    let
+      inherit (nixpkgs.lib) cleanSourceWith hasSuffix;
+      overlay = final: prev: {
+        cubical = final.agdaPackages.mkDerivation rec {
+          pname = "cubical";
+          version = "0.4";
+
+          src = cleanSourceWith {
+            filter = name: type:
+              !(hasSuffix ".nix" name)
+            ;
+            src = ./.;
+          };
+
+
+          LC_ALL = "C.UTF-8";
+
+          preConfigure = ''export AGDA_EXEC=agda'';
```

Comment:
```suggestion
```

This doesn't seem to be needed anymore as `AGDA_EXEC` defaults to `agda -W error`.